template.js

```diff
@@ -287,23 +387,18 @@
     // This is because the native import node creates the right document owned
     // subtree and `fixClonedDom` inserts cloned templates into this subtree,
     // thus updating the owner doc.
-    Document.prototype.importNode = function(element, deep) {
+    var importNode = Document.prototype.importNode = function importNode(element, deep) {
+      deep = deep || false;
       if (element.localName === TEMPLATE_TAG) {
         return PolyfilledHTMLTemplateElement._cloneNode(element, deep);
       } else {
-        var dom = Native_importNode.call(this, element, deep);
+        var dom = capturedImportNode.call(this, element, deep);
         if (deep) {
-          PolyfilledHTMLTemplateElement.fixClonedDom(dom, element);
+          PolyfilledHTMLTemplateElement._fixClonedDom(dom, element);
         }
         return dom;
       }
     };
-
-    if (needsCloning) {
```

Comment:
Why is this removed?

Reply:
Not needed since I'm overriding `Node.prototype.cloneNode` entirely.